### ISSUE TITLE
pump and drainer e2e bug

### ISSUE THREAD

**Issue by weekface** [pingcap/tidb-operator#743]
https://github.com/pingcap/tidb-operator/blob/c348598daf7cc15fa6482f8642e90ab2a4193b64/tests/actions.go#L2464-L2466

this struct needs json tag.

```
type pumpStatus struct {
	StatusMap map[string]*nodeStatus
}

```
should be:

```
type pumpStatus struct {
	StatusMap map[string]*nodeStatus `json:"status"`
}
```

**Comment by weekface** [comment]
@shuijing198799 PTAL

**Comment by shuijing198799** [comment]
> @shuijing198799 PTAL

This issue is fixed in #764 PTAL
